homeassistant/components/zwave_js/services.py

```diff
@@ -135,7 +135,9 @@ def validate_multicast_nodes(val: dict[str, Any]) -> dict[str, Any]:
         @callback
         def validate_entities(val: dict[str, Any]) -> dict[str, Any]:
             """Validate entities exist and are from the zwave_js platform."""
-            val[ATTR_ENTITY_ID] = expand_entity_ids(self._hass, val[ATTR_ENTITY_ID])
+            val[ATTR_ENTITY_ID] = expand_entity_ids(
```

Comment:
what is the use case to allow to sent in group entities for light/switch/cover etc to a Z-Wave JS service? I wouldn't expect that to work because it's a new entity.

Reply:
If you have a light group of z-wave lights, and you want to set a config parameter or invoke the CC API on all of them in parallel, you would not be able to use the light group entity to do it before this change. @kpine do you have any additional use cases you can describe? I think this PR came out of a conversation you and I were having

Reply:
Your recollection is correct. There was simply an observation that for the zwave-js services, helper groups aren't being expanded by the integration, whereas `group` entities are being expanded (are those "old style" groups?) by the integration. A good example use-case would be to use the multicast service to turn on a light group, instead of having to list each light entity individually. HA Core does not provide a way to do this. Otherwise, it just makes the behavior of the two groups consistent. As a user, not concerned with implementation details, I'd expect it to just work, because why not?

That said, it's currently "easy" (if you know how) to workaround by expanding the group manually in a template with `expand()`, although the Group integration and Templating docs don't state whether this is actually supported for helper groups. It only mentions `expand()`  for `group` entities (old style groups?). I just happened to notice it worked with light groups by experimentation.

Reply:
It is supported in the `expand` function, that's where I got the solution for this. We should probably update the docs